src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java

```diff
@@ -373,6 +375,28 @@ public void testForceCalling() {
         }
     }
 
+    // test that the dont-use-soft-clips option actually does something
+    @Test
+    public void testDontUseSoftClips() {
+        Utils.resetRandomGenerator();
+        final File tumor = new File(NA12878_20_21_WGS_bam);
+        final int start = 10050000;
+
+        final SimpleInterval interval = new SimpleInterval("20", start, start + 25000);
+
+        final File calls1 = createTempFile("unfiltered", ".vcf");
+        runMutect2(tumor, calls1, interval.toString(), b37Reference, Optional.of(GNOMAD));
+
+        final File calls2 = createTempFile("unfiltered", ".vcf");
+        runMutect2(tumor, calls2, interval.toString(), b37Reference, Optional.of(GNOMAD),
+                args -> args.addFlag(AssemblyBasedCallerArgumentCollection.DONT_USE_SOFT_CLIPPED_BASES_LONG_NAME));
+
+        final List<VariantContext> indelsWithSoftClips = VariantContextTestUtils.streamVcf(calls1).filter(VariantContext::isIndel).collect(Collectors.toList());
+        final List<VariantContext> indelsWithoutSoftClips = VariantContextTestUtils.streamVcf(calls2).filter(VariantContext::isIndel).collect(Collectors.toList());
+
+        Assert.assertTrue(indelsWithoutSoftClips.size() < indelsWithSoftClips.size());
```

Comment:
Can we be more specific about which variants are called/not called when the option is set/unset?
I'd hate for this to pass by chance, which is almost has a 50/50 chance of, even if it's not doing what we expect.